examples/simpleserver/cmd.go

```diff
@@ -32,7 +31,7 @@ func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error
 			ArmModel: "ur5e",
 		},
 	}
-	arm1, err := fake.NewArm(&inject.Robot{}, cfg, logger)
+	arm1, err := fake.NewArm(cfg, logger)
```

Comment:
this directory and specifically, `cmd.go` will not be ran as part of a CI check.
[nit]: validate that this is not broken

Reply:
`cmd.go` executes successfully, and I can confirm the `arm1` is a resource in `myRobot`.